metaflow/plugins/kfp/kfp.py

```diff
@@ -998,23 +748,67 @@ def build_kfp_dag(
                     METAFLOW_USER=METAFLOW_USER,
                 )
                 metaflow_run_id = f"kfp-{dsl.RUN_ID_PLACEHOLDER}"
-                step_op_args = dict(
-                    cmd_template=kfp_component.cmd_template,
-                    metaflow_run_id=metaflow_run_id,
-                    metaflow_configs=metaflow_configs,
-                    passed_in_split_indexes=passed_in_split_indexes,
-                    preceding_component_inputs=preceding_component_inputs,
-                    preceding_component_outputs=kfp_component.preceding_component_outputs,
-                    flow_parameters_json=flow_parameters_json
-                    if node.name == "start"
-                    else None,
+
+                command = [
+                    "bash",
+                    "-ec",
+                    kfp_component.init_cmd
```

Comment:
Seems like this `init_cmd` (at least for the components in the before/after dumps above) is just `cd /opt/zillow/aip_workflow_example`. I'm wondering if this is something we can remove too, then we'd be able to remove the `bash -ec` too if we remove the explicit `exit` below too.

I'm guessing we can explicitly set the `/opt/zillow/aip_workflow_example` directory via a parameter, something like:

```python -m <module> --metaflow-context /opt/zillow/aip_workflow_example ...` (Sorry not sure the name of the parameter or if there is one!)

Reply:
The thing is, `init_cmd` is longer in the case of `--s3-code-package` and it not a simple `cd` command unfortunately. We were thinking about having a template for that code in the images, but I don't think that will be part of this PR effort.

Reply:
Cool thanks for clarifying!